Firebase/Messaging/FIRMessagingRmqManager.m

```diff
@@ -106,6 +119,8 @@
 
 @interface FIRMessagingRmqManager () {
   sqlite3 *_database;
```

Comment:
Though it's not part of this change, make sure the database is opened with `SQLITE_OPEN_FULLMUTEX`.

Reply:
We put all database operations in serial queue already.

Reply:
SGTM, I had overlooked that bit